spec/book_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'spec_helper'
+
+# require 'date'
+
+describe Book do
+  before :each do
+    @book = Book.new('title', 'Publisher', 'good')
+  end
+
+  describe '#new' do
+    it 'takes 5 parameters and returns a Book object' do
+      expect(@book).to be_an_instance_of Book
+    end
+  end
+
+  describe '#publisher' do
+    it 'returns the correct publisher' do
+      expect(@book.publisher).to eql 'Publisher'
+    end
+  end
+
+  describe '#cover_state' do
+    it 'returns the correct publisher' do
+      expect(@book.cover_state).to eql 'good'
+    end
+  end
```

Comment:
Team member #1

Based on the requirement you need to write tests for your all methods. You did a good job of adding a test for `#cover_state`. I would also suggest you write a test for `can_be_archived?()` as well. 

> So, in this test:
> should return true if the parent's method  returns true OR if cover_state equals to "bad"
> otherwise, it should return false